File: Example/Tests/NYTPhotoTransitionAnimatorTests.m
//
//  NYTPhotoTransitionAnimatorTests.m
//  NYTPhotoViewer
//
//  Created by Brian Capps on 2/26/15.
//  Copyright (c) 2015 NYTimes. All rights reserved.
//

@import UIKit;
@import XCTest;

#import <NYTPhotoViewer/NYTPhotoTransitionAnimator.h>

@interface NYTPhotoTransitionAnimatorTests : XCTestCase

@end

@implementation NYTPhotoTransitionAnimatorTests

- (void)testNewAnimationViewReturnsNilWhenPassedNil {
    XCTAssertNil([NYTPhotoTransitionAnimator newAnimationViewFromView:nil]);
}

@end

File: Example/NYTPhotoViewer-SwiftTests/NYTPhotoTransitionAnimatorTests.swift
//
//  NYTPhotoTransitionAnimatorTests.swift
//  NYTPhotoViewer
//
//  Created by Mark Keefe on 3/21/15.
//  Copyright (c) 2015 The New York Times. All rights reserved.
//

import XCTest

class NYTPhotoTransitionAnimatorTests: XCTestCase {

    func testNewAnimationViewReturnsNilWhenPassedNil() {
        XCTAssertNil(NYTPhotoTransitionAnimator.newAnimationViewFromView(nil))
    }
}
